addons/nuxeo-retention/nuxeo-retention-core/src/main/java/org/nuxeo/retention/RetentionConstants.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * (C) Copyright 2019 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Guillaume RENARD
+ */
+package org.nuxeo.retention;
+
+import org.apache.commons.lang3.time.FastDateFormat;
+import org.nuxeo.ecm.core.api.CoreSession;
+import org.nuxeo.retention.adapters.RetentionRule;
+
+/**
+ * @since 11.1
+ */
+public class RetentionConstants {
+
```

Comment:
as it is a constants class can you add a private constructor to avoid / prevents any instantiation :

```
// Prevents instantiation
private RetentionConstants() {
}
```

Reply:
The class could also be final.